package.json

```diff
@@ -99,7 +99,7 @@
     "@nrwl/web": "13.4.6",
     "@nrwl/workspace": "13.4.6",
     "@tailwindcss/aspect-ratio": "0.4.0",
-    "@tailwindcss/line-clamp": "0.3.0",
+    "@tailwindcss/line-clamp": "^0.3.1",
```

Comment:
Everything looks good except it looks like we need to pin these. I think you'll need to run `npm i` for each of these without the `^` (so `npm i @tailwindcss/line-clamp@0.3.1` in this case) in order to make sure package.lock is updated accordingly.

Reply:
I think these are dev deps, so `npm i --save-dev package-name@1.2.3` will be needed.